hash/ibm5170_cdrom.xml

```diff
@@ -64,14 +64,62 @@ license:CC0
 		</part>
 	</software>
 
+	<!-- Source: http://redump.org/disc/34886/ -->
+	<!-- <rom name="Blues Brothers, The (Europe).cue" size="94" crc="22379e3c" sha1="aaa704acbe4dcc90e8153f81227f00a83cbdaadd"/> -->
+	<!-- <rom name="Blues Brothers, The (Europe).bin" size="8979936" crc="346e8d8f" sha1="3b775aedf36fcbc33b6a04159d2c4887ec3d611d"/> -->
+	<software name="5plus1_10">
+		<description>5 Plus One: Pack 10 - The Blues Brothers (compilation)</description>
+		<year>1994</year>
+		<publisher>Prism Leisure Corporation PLC</publisher>
+		<notes><![CDATA[CD-ROM includes
+Complete Game: "The Blues Brothers"
+Demo/Shareware: "Bodycount", "Zone 66", "Save Our Pizzas", "Dangerous Dave", "Casino Slots"
+]]></notes>
+		<info name="region" value="Europe" />
+		<info name="usage" value="DOS" />
+
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Blues Brothers, The (Europe)" sha1="a9a4f1071e7ed928e73535d9bffcd11ee47ac56f" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Source: http://redump.org/disc/35748/ -->
+	<!-- <rom name="TV Sports Football (Europe).cue" size="93" crc="64a640a0" sha1="fbc6fcb695b38c0792312a23ae3c79a4aab6ee7b"/> -->
+	<!-- <rom name="TV Sports Football (Europe).bin" size="6905472" crc="11800913" sha1="6660a474a49f71485cc5ee802dafca90736c53fc"/> -->
+	<software name="5plus1_25">
+		<description>5 Plus One: Pack 25 - TV Sports Football (compilation)</description>
+		<year>1994</year>
+		<publisher>Prism Leisure Corporation PLC</publisher>
+		<notes><![CDATA[CD-ROM includes
+Complete Game: "TV Sports Football"
+Demo/Shareware: "Ranger Fox", "Monster Bash", "Onesimus", "Turoid", "Peristroka"
+]]></notes>
+		<info name="region" value="Europe" />
+		<info name="usage" value="DOS" />
+
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="TV Sports Football (Europe)" sha1="4765a2400d6e480a5656a7889331002ee7e910a6" />
+			</diskarea>
+		</part>
+	</software>
+
 	<!-- DOS (with Adobe Acrobat 2.1 for Windows) -->
 	<!-- disc label: pack34 -->
 	<!-- Super Tetris has manual copy protection -->
 	<!-- Other games (shareware): Green, Aldo II Adventure, Phylox, Draw Poker, Space Miner -->
-	<software name="5p1stet">
-		<description>5 Plus One - Super Tetris (Compilation)</description>
+	<software name="5plus1_34">
+		<description>5 Plus One: Pack 34 - Super Tetris (compilation)</description>
 		<year>1996</year>
 		<publisher>Prism Leisure Corporation PLC</publisher>
+		<notes><![CDATA[CD-ROM includes
+Complete Game: "Super Tetris"
+Demo/Shareware: "Green", "Aldo II Adventure", "Phylox", "Draw Poker", "Space Miner"
+]]></notes>
+		<info name="region" value="Europe" />
+		<info name="usage" value="DOS" />
```

Comment:
In general usage field is more for usability notes in order to make the SW to run, in this case `Super Tetris sports manual copy protection` as the comment above states. Given that the whole file should really be renamed to something like a generic `cdrom_iso9660.xml` I don't completely mind the change tho, but maybe use a different info entity for indicating the intended OS for the time being?

(Some of the stuff living in this file can run on other archs as well, particularly stuff that supports Windows 95/98 such as PC-98 and FM Towns, and iirc hybrid disks lives here as well ...)

Reply:
> In general usage field is more for usability notes in order to make the SW to run, in this case `Super Tetris sports manual copy protection` as the comment above states. Given that the whole file should really be renamed to something like a generic `cdrom_iso9660.xml` I don't completely mind the change tho, but maybe use a different info entity for indicating the intended OS for the time being?
> 
> (Some of the stuff living in this file can run on other archs as well, particularly stuff that supports Windows 95/98 such as PC-98 and FM Towns, and iirc hybrid multi OS disks lives here as well ...)

I can add an info platform tag, something like:
`<info name="platform" value="Windows 95" />`
or
`<info name="platform" value="DOS" />`
Is this way more understandable?

Reply:
Thought a bit, I believe `<sharedfeat name="platform" value="DOS" />` is a better apt property, I guess it won't filter out entries with current codebase.

Reply:
I'll wait a bit and see if other suggestions come up, before I make any change.

Reply:
Is there a way to specify multple `platform` values? There are some CDs that are hybrid Mac/DOS CDs, for example. Even if these softlists are not yet attached to multiple systems, it might make sense to already start tracking those?

The `platform` tag would also be useful for other CD ROMs that have software for SGI, Sun and Windows for example

Reply:
I've added multiple platform comments to CDs, but only as commented comments, not with XML tags. I would like a way to have this info tracked there, too.

Reply:
> Thought a bit, I believe `<sharedfeat name="platform" value="DOS" />` is a better apt property, I guess it won't filter out entries with current codebase.

I've done the changes with your suggested tag.

Reply:
> I've added multiple platform comments to CDs, but only as commented comments, not with XML tags. I would like a way to have this info tracked there, too.

In time, I'll do it for the rest.

Reply:
> Is there a way to specify multple `platform` values? There are some CDs that are hybrid Mac/DOS CDs, for example. Even if these softlists are not yet attached to multiple systems, it might make sense to already start tracking those?

Just adding comma separators will work, saturn.xml has NTSC-K software which incidentally works on other regions too.